internal/app/handler/handler.go

```diff
@@ -15,14 +16,15 @@ import (
 const (
 	ContentType     = "Content-Type"
 	ContentTypeText = "text/plain"
+	ContentTypeJSON = "application/json"
 )
 
 type Handler struct {
-	service service.URLShortener
-	config  config.Config
+	service *service.URLShortener
```

Comment:
```suggestion
	service service.URLShortener
```

Reply:
I did not quite get this comment. May be because in the past it was an interface. Now service.URLShortener has been now refactored to a struct, so i pass it by pointer.

Reply:
It should be an interface.